benchmarks/csharp/Program.cs

```diff
@@ -21,216 +19,192 @@ private enum ChosenAction { GET_NON_EXISTING, GET_EXISTING, SET };
     public class CommandLineOptions
     {
         [Option('r', "resultsFile", Required = false, HelpText = "Set the file to which the JSON results are written.")]
-        public string resultsFile { get; set; } = "../results/csharp-results.json";
+        public string ResultsFile { get; set; } = "../results/csharp-results.json";
 
         [Option('d', "dataSize", Required = false, HelpText = "The size of the sent data in bytes.")]
-        public int dataSize { get; set; } = 100;
+        public int DataSize { get; set; } = 100;
 
         [Option('c', "concurrentTasks", Required = false, HelpText = "The number of concurrent operations to perform.", Default = new[] { 1, 10, 100, 1000 })]
-        public IEnumerable<int> concurrentTasks { get; set; } = Enumerable.Empty<int>();
+        public IEnumerable<int> ConcurrentTasks { get; set; } = [];
```

Comment:
This is not compatible with dotnet 6

Reply:
Maybe I'm mistaken, but when I tried to do that change, I got a warning about compatibility.

Reply:
It is compatible with dotnet 6. I think your issue was with C# language version. Which I uploaded

Reply:
Excellent, please ignore these comments then.
I need to update things on my workstation.